### PR TITLE
Adds content to website 

### PR DESCRIPTION
**Pull request by arkinem**
PAGE CHANGES PREVIEW: https://kind-meitner-56bfd3.netlify.com/

### CODE REVIEW

**Comment by arkinem** [comment]
PAGE CHANGES PREVIEW: https://kind-meitner-56bfd3.netlify.com/

File: index.html [hunk 1]
```diff
@@ -273,7 +273,7 @@ <h3>Event Organisers:</h3>
           <div>
             Bruno Boto, Faisal Hoque, Mateusz Kitlinski, Belgin <br />
```
Comment (brunoboto96) [inline index.html]: remove peeps

File: index.html [hunk 2]
```diff
@@ -1,161 +1,329 @@
...
+        </div>
+        <div class="col-6 col-md">
+          <h3>Event Organisers:</h3>
+          <img
+            src="https://www.loveluton.org.uk/wp-content/uploads/University-of-Bedfordshire.png"
+            width="100px"
+          />
+          <div>
+            Bruno Boto, Faisal Hoque, Mateusz Kitlinski, Belgin <br />
+            Seitmola, Marian Terchila, Blazej Golinski, <br />
```
Comment (brunoboto96) [inline index.html]: remove peeps

File: index.html [hunk 3]
```diff
@@ -1,161 +1,329 @@
...
+
+    <footer class="jumbotron mb-0">
+      <hr />
+      <div class="row">
+        <div class="col-6 col-md">
+          <h3>Sponsored by:</h3>
+          <img
+            src="https://wp-cdn.ziftsolutions.com/wp-content/uploads/2017/12/ZiftSolutions.svg"
+            style="width: 30vw; padding: 5%"
+          />
```
Comment (brunoboto96) [inline index.html:273]: needs link

File: index.html [hunk 4]
```diff
@@ -1,161 +1,329 @@
...
+        </div>
+        <div class="col-6 col-md">
+          <h3>Event Organisers:</h3>
+          <img
+            src="https://www.loveluton.org.uk/wp-content/uploads/University-of-Bedfordshire.png"
```
Comment (brunoboto96) [inline index.html:279]: link

File: index.html [hunk 5]
```diff
@@ -1,161 +1,329 @@
...
+      >
+        <div class="my-3 py-3">
+          <h2 class="display-5">Support</h2>
+          <p class="lead">Click the Mail icon and ask us anything</p>
+        </div>
+        <div
+          class="bg-light box-shadow mx-auto"
+          style="width: 80%; height: 300px; border-radius: 21px 21px 0 0;"
+        >
+          <a href="mailto:support@gmail.com"
```
Comment (brunoboto96) [inline index.html]: csthackathon.beds@gmail.com

File: index.html [hunk 6]
```diff
@@ -1,161 +1,329 @@
...
+            <div class="workshop-container">
+              <div class="workshop-item datetime">Sat 13:00</div>
+              <div class="workshop-item workshop-name">3D Printing</div>
+              <div class="workshop-item">Ana and Bruno Boto</div>
+            </div>
+            <div class="workshop-container">
+              <div class="workshop-item datetime">Sat 15:30</div>
+              <div class="workshop-item workshop-name">Virtual Reality</div>
+              <div class="workshop-item">Mathew Barnes</div>
+            </div>
```
Comment (brunoboto96) [inline index.html:116]: Could add CTF here

File: index.html [hunk 7]
```diff
@@ -1,161 +1,330 @@
...
+          <div class="workshops-container">
+            <div class="workshop-container">
+              <div class="workshop-item datetime">Sat 10:00</div>
+              <div class="workshop-item workshop-name">Design Thinking</div>
+              <div class="workshop-item">Bruno Boto</div>
+            </div>
+            <div class="workshop-container">
+              <div class="workshop-item datetime">Sat 13:00</div>
+              <div class="workshop-item workshop-name">3D Printing</div>
+              <div class="workshop-item">Bruno Boto</div>
```
Comment (brunoboto96) [inline index.html:110]: removed anna

**Review by brunoboto96 — APPROVED** [review]
Cool stuff

File: index.html (hunk 5 above)
Comment (brunoboto96) [reply index.html]: done

**Comment by brunoboto96** [comment]
merged into master and server

**Pull request merged**